backend/qemu.pm

```diff
@@ -868,7 +868,7 @@ sub start_qemu ($self) {
         }
         if ($vars->{NBF}) {
             die "Need variable WORKER_HOSTNAME\n" unless $vars->{WORKER_HOSTNAME};
-            sp('kernel', '/usr/share/qemu/ipxe.lkrn');
+            sp('kernel', '/usr/share/ipxe/ipxe.lkrn');
```

Comment:
I wonder if there should be a fallback? Since this effectively breaks existing setups that aren't relying on the packaged path

Reply:
Thanks for the kindly comment!
Yes, I added more information in description part, I think we should have to merge [MR-salt](https://gitlab.suse.de/openqa/salt-states-openqa/-/merge_requests/1016) at first, and make sure the workers install the package 'ipxe-bootimgs' at first before merging this PR.

Do we have any back-end service/process to do that?

Reply:
This should probably be added to `dependencies.yaml` since salt only helps with osd.

Reply:
This would then technically still break compatibility for all users of os-autoinst that use it not via our packaging. I guess it is ok because supposedly nobody uses this anyways. On the other hand, adding a fallback wouldn't hurt.

Reply:
```suggestion
            sp('kernel', -e '/usr/share/ipxe/ipxe.lkrn' ? '/usr/share/ipxe/ipxe.lkrn' : '/usr/share/qemu/ipxe.lkrn');
```

Reply:
Thanks much! fixed